projects/cds/src/config/default-cds-config.ts

```diff
@@ -0,0 +1,18 @@
+import { CdsConfig } from './cds-config';
+
+export const DEFAULT_CDS_CONFIG: CdsConfig = {
+  cds: {
+    tenant: '',
+    baseUrl: 'https://api.stage.context.cloud.sap',
+    endpoints: {
+      strategyProducts: '/strategy/${tenant}/strategies/${strategyId}/products',
+    },
+    profileTag: {
+      javascriptUrl:
+        'https://tag.static.us.context.cloud.sap/js/beta/profile-tag.js',
+      configUrl:
+        'https://tag.static.stage.context.cloud.sap/config/dfbb97b0-f4d7-11e9-9c99-2125ab7968c6',
```

Comment:
Just for the future reference, the UID should be removed before the final merge to `develop`. You can leave it for now, if it's more convenient for you to test / develop.

Reply:
I think Corey will probably update this configuration before merging to develop anyway